Description: Drop removed ControlGroup stanza
 Systemd >= 205 do not have this directive anymore.
 This is not a problem for us because the debian kernel does not have
 CONFIG_RT_GROUP_SCHED enabled, so rt budgets are not per-cgroup
Bug-Debian: 747568
--- a/rtkit-daemon.service.in
+++ b/rtkit-daemon.service.in
@@ -27,9 +27,5 @@ CapabilityBoundingSet=CAP_SYS_NICE CAP_D
 PrivateTmp=yes
 PrivateNetwork=yes
 
-# Work around the fact that the Linux currently doesn't assign any RT
-# budget to CPU control groups that have none configured explicitly
-ControlGroup=cpu:/
-
 [Install]
 WantedBy=graphical.target
